pkg/event/event.go

```diff
@@ -71,7 +71,10 @@ var LevelMap = map[config.EventType]config.Level{
 func New(objectMeta metaV1.ObjectMeta, object interface{}, eventType config.EventType, resource string) (Event, error) {
 	typeMeta := k8sutil.GetObjectTypeMetaData(object)
 	event := Event{
-		TypeMeta:   typeMeta,
```

Comment:
I know that we will fix that in the follow-up task. But let's remove that already so we will have a better diff with `internal/source/kubernetes/event/event.go` - in the rest of the code you can import that moved event instead.

Reply:
Isn't that dangerous to depend a public package to a plugin internals? I just wanted to wait until platform is removed, this event package is still used in a couple of places.

Reply:
it's not, actually it's a proper solution to import some parts from `internal` in `pkg`. If you want I can explain on 1:1 call